lib/generators/manageiq/plugin/plugin_generator.rb

```diff
@@ -85,6 +85,16 @@ def insert_manageiq_gem
         end
       GEMFILE
       inject_into_file Rails.root.join('Gemfile'), "\n#{data}\n", :after => "### providers\n"
+
+      bundler_d_dir       = Rails.root.join("bundler.d")
+      plugin_overrides_rb = bundler_d_dir.glob("*.rb").first || bundler_d_dir.join("plugins.rb")
+
+      data = "override_gem \"#{plugin_name}\", :path => \"#{destination_root}\"\n"
+      if plugin_overrides_rb.exist?
+        inject_into_file plugin_overrides_rb, data
+      else
+        plugin_overrides_rb.write(data)
+      end
```

Comment:
Simpler?

```suggestion
      FileUtils.touch(plugin_overrides_rb) unless plugin_overrides_rb.exist?
      inject_into_file plugin_overrides_rb, data
```

Reply:
Maybe?  It seemed like a waste to touch the file then write to it vs just writing it once